### ISSUE TITLE
Strato restarted

### ISSUE THREAD

**Issue by BoehniM** [fhorinek/BB#279]
During flight Strato restarted apparently and it seems I've lost some minutes of my flight. Would be great to get this fixed. 

[debug.log](https://github.com/fhorinek/BB/files/8638098/debug.log)

**Comment by ElDevelish** [comment]
Two logs simply sounds as if your auto landing (and then takeoff) got triggered because the settings are too sensitive? 
Can you post them?

**Comment by BoehniM** [comment]
Sure. Here we go:

**Comment by BoehniM** [comment]
Did you geht the Screenshots?
![Uploading 10FCBE90-9808-4118-AB02-14F28DB64BEC.jpeg…]()

**Comment by BoehniM** [comment]
![7DC198D4-05C6-4035-86FC-1CFB52EDD992](https://user-images.githubusercontent.com/87314009/167717325-ca6f3bb0-0149-4554-b503-b639992ac67e.jpeg)

**Comment by ElDevelish** [comment]
No, they were not uploaded before. And the probably most important setting is still missing. If you click the setting, there is an option "within time". For soaring, both altitude change and speed can easily drop below those; I simply increased the time to 1 min - I don't care if the flight triggers a minute too late and keeps running after landing for 60s. 
Try that maybe?
